Orange/tests/test_random_forest.py

```diff
@@ -68,15 +68,25 @@ def test_predict_numpy_reg(self):
         self.assertTrue(all(pred) > 0)
 
     def test_scorer(self):
-        data = Table('test4.tab')
+        data = Table('iris')
         learner = RandomForestLearner()
         scores = learner.score_data(data)
         self.assertEqual(len(scores), len(data.domain.attributes))
         self.assertNotEqual(sum(scores), 0)
+        self.assertEqual(['petal length', 'petal width'],
+                         sorted([data.domain.attributes[i].name
+                                 for i in np.argsort(scores)[-2:]]))
+
+        data = Table('housing')
+        learner = RandomForestRegressionLearner()
+        scores = learner.score_data(data)
+        self.assertEqual(['LSTAT', 'RM'],
+                         sorted([data.domain.attributes[i].name
+                                 for i in np.argsort(scores)[-2:]]))
 
     def test_scorer_feature(self):
         np.random.seed(42)
-        data = Table('test4.tab')
+        data = Table('iris')
```

Comment:
I don't remember. I think when I looked at test4.tab, I thought it too trivial a file to test the scorer more thoroughly. Will take back.